skeleton/aws/README.md

```diff
@@ -0,0 +1,84 @@
+## Deploy your infrastructure
```

Comment:
Should we move it to Wiki instead?

Reply:
The idea is to have it in the generated project. Though the generated project does not have a wiki (and I do not think we aim for that as we usually use the Wiki of the Web repository). 
So it seems good to me, that we provide this piece of info in the README file and then the user can move it where (s)he wants ;-)

Reply:
That makes sense 👍